eclair-core/src/main/scala/fr/acinq/eclair/balance/BalanceActor.scala

```diff
@@ -85,9 +85,20 @@ private class BalanceActor(context: ActorContext[Command],
       res match {
         case Success(result) =>
           log.info("current balance: total={} onchain.confirmed={} onchain.unconfirmed={} offchain={}", result.total.toDouble, result.onChain.confirmed.toDouble, result.onChain.unconfirmed.toDouble, result.offChain.total.toDouble)
-          log.debug("current balance details : {}", result)
-          if (result.onChain.confirmed < 1.millibtc) {
-            context.system.eventStream ! EventStream.Publish(NotifyNodeOperator(NotificationsLogger.Warning, s"on-chain confirmed balance is low (${result.onChain.confirmed.toMilliBtc}), eclair may not be able to guarantee funds safety in case channels force-close: you should add some utxos to your bitcoin wallet"))
+          log.debug("current balance details: {}", result)
+          // This is a very rough estimation of the fee we would need to pay for a force-close with 5 pending HTLCs at 100 sat/byte.
+          val perChannelFeeBumpingReserve = 0.5.millibtc
```

Comment:
Maybe arbitrary, but I would have set this in `sat`, because it's the unit we use to pay the fees. The `millibtc` unit makes sense for aggregated amounts in the monitoring console. 🤔

Reply:
0.5 millibtc = 5000 sat ?